$OpenBSD: patch-src_master_master-settings_c,v 1.13 2012/01/07 12:14:21 sthen Exp $

changed settings are also present in src/config/all-settings.c; this file
is rebuilt as part of the Dovecot build.

--- src/master/master-settings.c.orig	Thu Jan  5 21:36:15 2012
+++ src/master/master-settings.c	Thu Jan  5 21:36:40 2012
@@ -214,8 +214,8 @@ static const struct master_settings master_default_set
 	.protocols = "imap pop3 lmtp",
 	.listen = "*, ::",
 	.ssl = "yes:no:required",
-	.default_internal_user = "dovecot",
-	.default_login_user = "dovenull",
+	.default_internal_user = "_dovecot",
+	.default_login_user = "_dovenull",
 	.default_process_limit = 100,
 	.default_client_limit = 1000,
 	.default_idle_kill = 60,
